src/main/scala/mesosphere/marathon/upgrade/DeploymentDeathRow.scala

```diff
@@ -0,0 +1,18 @@
+package mesosphere.marathon.upgrade
+
+import mesosphere.marathon.Protos.MarathonTask
+
+private[upgrade] case class DeploymentDeathRow(runningTasks: Set[MarathonTask],
```

Comment:
If you expand this as suggested above, you could call it "ScalingCalculator" or something like this.